UI/RemoteISOScreen.cpp

```diff
@@ -268,19 +285,34 @@ static bool LoadGameList(const std::string &host, int port, std::vector<std::str
 	std::vector<std::string> items;
 	result.TakeAll(&listing);
 
-	SplitString(listing, '\n', items);
-	for (const std::string &item : items) {
-		if (!endsWithNoCase(item, ".cso") && !endsWithNoCase(item, ".iso") && !endsWithNoCase(item, ".pbp")) {
-			continue;
+	if (startsWith(responseHeaders[1],"Content-Type: text/plain")) {
```

Comment:
Definitely not a good idea to assume that `Content-Type` will always be the second header.  Also, should be case insensitive.  I'd think something more like:

```c++
std::string contentType;
for (const std::string &header : responseHeaders) {
    if (startsWithNoCase(header, "Content-Type:")) {
        contentType = header.substr(strlen("Content-Type:"));
        // Strip any whitespace (TODO: maybe move this to stringutil?)
        contentType.erase(0, contentType.find_first_not_of(" \t\r\n"))
        contentType.erase(contentType.find_last_not_of(" \t\r\n") + 1);
    }
}

// TODO: Technically, "TExt/hTml    ; chaRSet    =    Utf8" should pass, but "text/htmlese" should not.
// But unlikely that'll be an issue.
bool parseHtml = startsWithNoCase(contentType, "text/html");
bool parseText = startsWithNoCase(contentType, "text/plain");

if (parseText) {
    // PPSSPP server / plain text response listing of URIs.
} else if (parseHtml) {
    // Parse out URLs.  Warning: some webservers may use relative href values, others absolute.
} else {
    ERROR_LOG(FILESYS, "Unsupported Content-Type: %s", contentType.c_str());
    return false;
}
```